### ISSUE TITLE
Not working anymore in chrome extension v3 manifest

### ISSUE THREAD

**Issue by AndriusCTR** [ilinsky/jquery-xpath#27]
_"Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'"."_

In other words, the latest chrome extension platform V3 does not allow eval and thus can't use this great library any more.

Any solutions?

**Comment by ilinsky** [comment]
The library does not appear to be using eval function (why should it?). The minimized version however does use Function constructor, which may cause the problem. Can you try not-minimized version maybe and report here?

**Comment by AndriusCTR** [comment]
Thanks, yeah full version seems to work properly.

I wonder if it's possible to have minimized version without eval...

**Comment by ilinsky** [comment]
Yes, the base engine has just been revived, and the dependant libraries builds will soon be created and published.

**Comment by AndriusCTR** [comment]
Thanks

**Comment by bonjour-anuj** [comment]
Any update on this?

**Comment by ilinsky** [comment]
No update yet.

**Comment by Polyfun** [comment]
I can confirm that the unminimized jquery.xpath.js v0.3.1 does not violate unsafe-eval content-security-policy, but the minimized jquery.xpath.min.js v0.3.1 does (I took both versions from the repository here). I am using Visual Studio 2022. I then used the Bundler & Minifier from https://marketplace.visualstudio.com/items?itemName=Failwyn.BundlerMinifier64 to create a new jquery.xpath.min.js from jquery.xpath.js, and this new jquery.xpath.min.js does not violate unsafe-eval content-security-policy. So there is something wrong in how the official jquery.xpath.min.js v0.3.1 in the repository here has been generated - it  is not 100 % equivalent to the unminimized jquery.xpath.js.
